app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
```

Comment:
- [ ] When I start the project locally I'm getting an error:
<img width="1047" alt="Screenshot 2022-10-12 at 22 19 38" src="https://user-images.githubusercontent.com/77038610/195429446-b39ef711-4d6e-43ce-9615-a7beaf14c708.png">

to recreate the issue, you can clone the repository, and try to start it.

Reply:
Hello @ShoiraTa I used tailwind so you have to yarn install and yarn build:css